src/core/opamHash.mli

```diff
@@ -32,6 +32,9 @@ val of_string_opt: string -> t option
     "md5/d4/d41d8cd98f00b204e9800998ecf8427e", as a list *)
 val to_path: t -> string list
 
+(** Sorts the list from best hash (SHA512, ...) to weakest (..., MD5) *)
+val sort : t list -> t list
```

Comment:
It feels a bit surprising to have a reverse sort being called sort tbh but I guess that’s ok in opam lib. We’re not a generalist library after all.

Reply:
That's the idea :) Like @AltGr said, its a sort from the less weak to the most weak :D

Reply:
adjugé vendu! :D